packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts

```diff
@@ -347,6 +347,14 @@ export const getVideoReceiverStreamMqa = ({
     statsResults[mediaType][sendrecvType].keyFramesDecoded - lastKeyFramesDecoded || 0;
   videoReceiverStream.requestedKeyFrames =
     statsResults[mediaType][sendrecvType].totalPliCount - lastPliCount || 0;
+
+  videoReceiverStream.isActiveSpeaker =
+    statsResults[mediaType][sendrecvType].isActiveSpeaker ||
+    ((statsResults[mediaType][sendrecvType].lastActiveSpeakerTimestamp ?? 0) > 0 &&
+      performance.now() +
+        performance.timeOrigin -
```

Comment:
We do this `performance.now() + performance.timeOrigin` calculation a lot across the `isActiveSpeaker` and `isRequested` calculations. It may be worth having a helper function to do this calculation for us so we don't have to type this out every time.